File: chatwoot/csharp/src/OSEG.ChatwootExamples/DeleteAgentInInboxExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class DeleteAgentInInboxExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "USER_API_KEY");

        var deleteAgentInInboxRequest = new DeleteAgentInInboxRequest(
            inboxId: "inbox_id_string",
            userIds: [
            ]
        );

        try
        {
            new InboxesApi(config).DeleteAgentInInbox(
                accountId: 0,
                data: deleteAgentInInboxRequest
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling InboxesApi#DeleteAgentInInbox: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/DeleteAgentInInboxExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
});

const deleteAgentInInboxRequest: api.DeleteAgentInInboxRequest = {
  inbox_id: "inbox_id_string",
  user_ids: [
  ],
};

new api.InboxesApi(configuration).deleteAgentInInbox(
  0, // accountId
  deleteAgentInInboxRequest, // data
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling InboxesApi#deleteAgentInInbox:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
